lib/core/src/utils.rs

```diff
@@ -49,3 +52,38 @@ pub(crate) fn deserialize_tx_hex(tx_hex: &str) -> Result<Transaction> {
         |err| anyhow!("Could not deserialize transaction: {err:?}"),
     )?)?)
 }
+
+pub(crate) fn estimate_refund_fees(
+    swap: &SendSwap,
+    config: &Config,
+    output_address: &str,
+    is_lowball: bool,
+) -> Result<u64, PaymentError> {
+    let swap_script = swap.get_swap_script()?;
+    let electrum_config = ElectrumConfig::new(
+        config.network.into(),
+        &config.liquid_electrum_url,
+        true,
+        true,
+        100,
+    );
+    let swap_tx = boltz_client::LBtcSwapTxV2::new_refund(
+        swap_script,
+        &output_address.to_string(),
+        &electrum_config,
+        config.liquid_electrum_url.clone(),
+        swap.id.clone(),
+    )?;
+    let dummy_fees = Amount::from_sat(100);
+    let dummy_tx = swap_tx.sign_refund(&swap.get_refund_keypair()?, dummy_fees, None)?;
```

Comment:
Maybe there is a difference in the tx vsize in a cooperative vs non-cooperative refund that we should take into account...

I guess it's actually not easy to do without doing any of the cooperative part, I see in boltz-rust there is a TODO to hardcode the cooperative spend size
https://github.com/SatoshiPortal/boltz-rust/blob/6ba4f48bebfe5bc05ce65fa5f44c1cfd9c42ffce/src/swaps/liquid.rs#L1183-L1193